usr/local/www/guiconfig.inc

```diff
@@ -266,11 +266,15 @@ function do_input_validation($postdata, $reqdfields, $reqdfieldsn, &$input_error
 }
 
 function print_input_errors($input_errors) {
+    print_errors($input_errors, gettext('The following input errors were detected:'));
```

Comment:
I wanted a function to display a general error message. This seemed like the most efficient way to do that. What would you suggest?

Steve
--
Stephen Beaver | +1 (512) 646-4100;239 | sbeaver@netgate.com | www.netgate.com 

> On Mar 28, 2015, at 10:50 AM, SjonHortensius <notifications@github.com> wrote:
> 
> In usr/local/www/guiconfig.inc <https://github.com/SjonHortensius/pfsense/pull/42#discussion_r27345559>:
> 
> > @@ -266,11 +266,15 @@ function do_input_validation($postdata, $reqdfields, $reqdfieldsn, &$input_error
> >  }
> >  
> >  function print_input_errors($input_errors) {
> > +    print_errors($input_errors, gettext('The following input errors were detected:'));
> I don't see any reason for refactoring this, it didn't come from master either?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/SjonHortensius/pfsense/pull/42/files#r27345559>.
>